src/doc/man/cargo-metadata.md

```diff
@@ -50,6 +53,8 @@ The output has the following format:
                  Example: "registry+https://github.com/rust-lang/crates.io-index"
                - "git+URL" for git-based dependencies.
                  Example: "git+https://github.com/rust-lang/cargo?rev=5e85ba14aaa20f8133863373404cb0af69eeef2c#5e85ba14aaa20f8133863373404cb0af69eeef2c"
+               Consumers should generally see it as an opaque string. There is
+               no guarantee of the format across different versions of Cargo.
             */
             "source": null,
```

Comment:
Should this be duplicated down in `dependencies`?

Reply:
> …see description for the package source.

So maybe no? Could add if you prefer to.

Reply:
Can't believe I glossed over that.  nm